source/cypress/fixtures/templaterepo/cdk/packages/compliant-dynamodb-table/README.md

```diff
@@ -0,0 +1,11 @@
+# `compliant-dynamodb-table`
+
+> TODO: description
```

Comment:
TODO?

Reply:
This is a template snippet for the users. The TODO is for users to add the required code to demonstrate the API.